examples/text-classification/README.md

```diff
@@ -101,7 +103,6 @@ export GLUE_DIR=/path/to/glue
 export TASK_NAME=MNLI
 
 python run_glue_tpu.py \
```

Comment:
This script doesn't exist anymore right?

Reply:
Yes that's correct. Left it in the doc for now so keep a trace at the parameter set you used, for reference.